pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -920,7 +924,46 @@ public String extractStringValueFrom(ResultSet rs) throws SQLException {
           return extractBigDecimalString(rs, getSqlKey());
         }
       }, //
-      REERENCE_SLIDE_ID("reference_slide_id", "Reference Slide");
+      REERENCE_SLIDE_ID("reference_slide_id", "Reference Slide"), TARGET_CELL_RECOVERY("target_cell_recovery", "Target Cell Recovery") {
```

Comment:
Could add the terminal `//` after REFERENCE_SLIDE_ID to make it clearer the below function is for TARGET_CELL_RECOVERY (if you like)

Reply:
Did that and fixed the spelling on `REFERENCE_SLIDE_ID` too